packages/desktop-client/src/components/Modals.js

```diff
@@ -283,6 +288,7 @@ function Modals({
             modalProps={modalProps}
             month={options.month}
             actions={actions}
+            isGoalTemplatesEnabled={isGoalTemplatesEnabled}
```

Comment:
Why not push the hook calls all the way down to the component that actually uses them to change behavior?

Reply:
`loot-core` and `loot-design` components should not rely on redux state or external data sources. Basically those are our component libraries. For the most part we have been following this pattern so far.. but not everywhere.

Reply:
Ah ok. I’ve been tossing around the idea of merging them again — I believe loot-design was intended to be shared between web and React Native but now they should be safe to combine together.

Reply:
Good idea. It would actually be nice to merge it in.. and perhaps even get rid of some custom components and replace with react-aria? Or something else so we don't have to maintain all that logic.